bee-network/bee-autopeering/src/proto/service.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package proto;
+
+// Mapping between a service ID and its tuple network_address
+// e.g., map[autopeering:&{tcp, 198.51.100.1:80}]
+message ServiceMap {
+  map<string, NetworkAddress> map = 1;
+}
+  
+// The service type (e.g., tcp, upd) and the address (e.g., 198.51.100.1:80)
```

Comment:
I feel like the comment is a bit off given the content or is it me ?
serbice type -> network
address -> port

Reply:
You are right...it feels off, but I am only partly to blame here: https://github.com/iotaledger/hive.go/blob/master/autopeering/peer/service/proto/service.proto

Side note:
Ideally I wouldn't have to c&p the proto files at all, and they would reside under a common `github.com/iotaledger/protos.git` repo, Hornet/Hive and Bee could then ideally use identical files, but the go version contains an extra line that I had to remove. I don't know if it's absolutely necessary in Go or if the Rust code could just ignore that. Would have to try.

Reply:
As explained, I have not defined this interface, or written those files. We need to coordinate changes with people outside of Bee first, but this shouldn't block this PR as it's just about comments.

Reply:
I understand the concern but at the same time, merging incorrect information just because it's already incorrect somewhere else seems weird to me. I was just suggesting fixing the comments.